src/Settings.php

```diff
@@ -164,6 +164,20 @@ public function __construct()
                         ),
                     ],
 
+                    [
+                        'name' => __('Post Revisions', 'planet4-master-theme-backend'),
+                        'id' => 'revisions_to_keep',
+                        'type' => 'text',
+                        'attributes' => [
+                            'type' => 'number',
+                        ],
+                        'default' => 5,
+                        'desc' => __(
+                            'This is the maximum of revisions to store into the database.',
```

Comment:
Minort re-phrasing suggestion: `'Maximum number of revisions to store for each post.'`